src/theme/ItaliaTheme/_main.scss

```diff
@@ -37,6 +37,7 @@
 }
 
 a {
+  display: inline-block; //per avere il bordo tutto unito al focus di un link, ad esempio nelle card
```

Comment:
Non sono sicuro sia ottimale metterlo su tutto il sito, qualsiasi tipo di link. Abbiamo problemi con i blocchi listing, le card etc, sistemerei solo lì. Questo ha effetto sui link nei testi, sui link in card più complesse che sono interamente cliccabili, sulle icone, insomma potrebbe avere effetto su cose non previste e rischiamo regressioni, quindi se possibile eviterei.